stubs/README.md

```diff
@@ -0,0 +1,2 @@
+
+Flasher stubs from https://github.com/espressif/esptool/tree/master/esptool/targets/stub_flasher
```

Comment:
I think it should be noted here that the stubs are GPLv2 licensed. Not sure if including them will affect the overall license of this repository.